### PR TITLE
main.go: introduce select-order-by-desc

### PR DESCRIPTION
**Pull request by dkropachev**
Scylla is supporting order by desc syntax,
we need to have ability to add it to the select queries.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -91,9 +92,9 @@ var (
 	startTime                time.Time
 	stopAll                  uint32
 	measureLatency           bool
-	hdrLatencyFile  string
-	hdrLatencyUnits string
-	validateData    bool
+	hdrLatencyFile           string
```
Comment (haaawk) [inline main.go:95]: Formatting cleanups should be in separate commit. All of them.

**Review by haaawk — APPROVED** [review]
LGTM in general. Only unrelated cleanups should be in a separate commit. @mmatczuk Could you please review the Golang side of this change?

File: main.go (hunk 1 above)
Comment (dkropachev) [reply main.go:95]: Done

**Review by haaawk — APPROVED** [review]

**Comment by roydahan** [comment]
Thank you all.
